assets/js/modules/analytics/components/settings/OptionalSettingsView.js

```diff
@@ -27,18 +27,25 @@ import { Fragment } from '@wordpress/element';
  */
 import Data from 'googlesitekit-data';
 import DisplaySetting from '../../../../components/DisplaySetting';
+import { CORE_MODULES } from '../../../../googlesitekit/modules/datastore/constants';
 import { CORE_SITE } from '../../../../googlesitekit/datastore/site/constants';
 import { trackingExclusionLabels } from '../common/TrackingExclusionSwitches';
 import { MODULES_ANALYTICS } from '../../datastore/constants';
+import { MODULES_ANALYTICS_4 } from '../../../analytics-4/datastore/constants';
+import { useFeature } from '../../../../hooks/useFeature';
 const { useSelect } = Data;
 
 export default function OptionalSettingsView() {
+	const ga4ReportingEnabled = useFeature( 'ga4Reporting' );
 	const useSnippet = useSelect( ( select ) =>
 		select( MODULES_ANALYTICS ).getUseSnippet()
 	);
 	const canUseSnippet = useSelect( ( select ) =>
 		select( MODULES_ANALYTICS ).getCanUseSnippet()
 	);
+	const isAnalyticsConnected = useSelect( ( select ) =>
+		select( CORE_MODULES ).isModuleConnected( 'analytics' )
+	);
```

Comment:
This seems to work as expected regardless, but it's important to keep in mind that the connected state of UA is not accurate from this selector when GA4 reporting is enabled.

Reply:
Thanks, despite things working I've changed this to use a more explicit check for the UA property being connected, to be on the safe side.